README.md

```diff
@@ -36,6 +36,38 @@ _Note: The Search UI is in beta. We do not recommend production use._
 Node: 8.10
 NPM: 6.4
 
+### Mono-repo explanation
+
+This repository is maintained as a Monorepo using [Lerna](https://lernajs.io/).
+
+Lerna configuration is contained in `lerna.json`.
+
+`/packages` - Contains publishable search-ui npm packages.
+`/examples` - Contains non-publishable examples of search-ui usage. They are declared
+as "packages" in `lerna.json` so that `npx lerna bootstrap` will automatically wire up the
+examples to local dependencies.
+
+Because all examples are declared as "private", when running lerna commands other than bootstrap, (like `publish` and
+`test`), the `--no-private` flag should be appended.
+
+Dependencies are declared in a package.json hierarchy.
+
+- / package.json - Dependencies for repo tooling, like `husky` and `lerna`.
+- /packages package.json - Common dev dependencies for all Search UI npm packages. Any dev depenency that does not need to be called directly in a package level npm command
+  can be delcared here.
+- /packages/{package_name} package.json - Package specific dependencies.
+
+Note that we do not encourage "hoisting" dependencies through lerna. This WILL
+cause the examples applications to error out from dependency version conflicts.
+
+### Installing dependencies
+
+```shell
+npm install
+(cd packages && npm install)
```

Comment:
Not sure what the parenthesis means here. Should I run these commands separately?

After getting an error, I ran `npm install` inside of packages and it worked.

Reply:
It's meant to be a single shell command, literally run as `(cd packages && npm install)`.

Reply:
Meaning, it's meant to be copied and pasted like that directly into your terminal and run

Reply:
Ah, ok. I haven't seen the parentheses used a lot before.